### ISSUE TITLE
Submit strings collection to Crowdin

### ISSUE THREAD

**Issue by tsusanka** [trezor/trezor-firmware#1019]
After #991 is done let's periodically upload the collected strings to Crowdin for translations.

---

Part of #987. Needs #991.

**Comment by Hannsek** [comment]
@grdddj this is done, right?

**Comment by grdddj** [comment]
Yes, the translations are in `Crowdin` - done manually

**Comment by obrusvit** [comment]
Closing in favor of https://github.com/trezor/trezor-firmware/issues/4867
